app/javascript/modules/listingDetails/ListingDetailsAside.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import { RailsListing } from "../listings/SharedHelpers"
+import { ListingDetailsInfoSession } from "../listingDetailsAside/ListingDetailsInfoSession"
+import { ListingDetailsApply } from "../listingDetailsAside/ListingDetailsApply"
+import { ListingDetailsProcess } from "../listingDetailsAside/ListingDetailsProcess"
+import { isOpen } from "../../util/listingUtil"
+import { ListingDetailsApplicationDate } from "../listingDetailsAside/ListingDetailsApplicationDate"
+import { ListingDetailItem } from "@bloom-housing/ui-components"
+
+export interface ListingDetailsSidebarProps {
+  listing: RailsListing
+}
+
+export const ListingDetailsAside = ({ listing }: ListingDetailsSidebarProps) => {
+  const isApplicationOpen = isOpen(listing)
+
+  return (
+    <ListingDetailItem
+      imageAlt={""}
+      imageSrc={""}
+      title={"Process"}
+      subtitle={"Process subtitle"}
+      hideHeader={true}
+      desktopClass="header-hidden"
+    >
+      <aside className="w-full static md:absolute md:right-0 md:w-1/3 md:top-0 sm:w-2/3 md:ml-2 h-full md:border border-solid bg-white">
+        <div className="hidden md:block">
+          <ListingDetailsApplicationDate listing={listing} />
+          {isApplicationOpen && <ListingDetailsInfoSession listing={listing} />}
```

Comment:
Should the parent or the component itself decide if it should display. Thoughts?